pydrive2/files.py

```diff
@@ -373,6 +382,8 @@ def GetContentIOBuffer(
         :type remove_bom: bool
         :param chunksize: default read()/iter() chunksize.
         :type chunksize: int
+        :param acknowledge_abuse: Allow abusive files.
```

Comment:
same, something like this `Acknowledging the risk and download file identified as abusive`